src/Components/Artwork/__tests__/Details.jest.tsx

```diff
@@ -579,4 +590,87 @@ const artworkInAuction: Details_Test_Query$rawResponse["artwork"] = {
     lotWatcherCount: null,
     partnerOffer: null,
   },
+  consignmentSubmission: null,
+  isListed: false,
+}
+
+const myCollectionArtworkSubmission: Details_Test_Query$rawResponse["artwork"] = {
```

Comment:
Because the data contains an artwork 🤷 

```suggestion
const submittedMyCollectionArtwork: Details_Test_Query$rawResponse["artwork"] = {
```